travis/tests-miniwdl.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -ex -o pipefail
+
+mkdir -p workflows
+cp -r test workflows/
+cd workflows
+
+miniwdl run_self_test
+
+for workflow in ../pipes/WDL/workflows/*.wdl; do
+	workflow_name=$(basename $workflow .wdl)
+	input_json="test/input/WDL/test_inputs-$workflow_name-local.json"
+	expected_output_json="test/input/WDL/test_outputs-$workflow_name-local.json"
+	if [ -f $input_json ]; then
+		date
+		echo "Executing $workflow_name using miniWDL on local instance"
+		miniwdl run -i $input_json -d $workflow_name/. --error-json $workflow
+		if [ -f $workflow_name/outputs.json ]; then
```

Comment:
Thoughts on trapping termination signals to auto-cleanup outputs (may be nice for testing locally), [as in the womtool test script](https://github.com/broadinstitute/viral-pipelines/blob/c41979f6fd62bb7afbed4245c58322f23c33a4d7/travis/validate-wdl-womtool.sh#L7-L14)?

Reply:
Good thought. I've personally had a little trouble getting miniwdl running on my Mac so that's why I haven't pursued that, let me play a bit here.